src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -1380,9 +1382,9 @@ MulticopterPositionControl::task_main()
 					_att_sp_pub = orb_advertise(_attitude_setpoint_id, &_att_sp);
 				}
 
-			} else if (_control_mode.flag_control_manual_enabled
-					&& _vehicle_land_detected.landed) {
-				/* don't run controller when landed */
+			} else if (_vehicle_land_detected.landed
```

Comment:
@LorenzMeier It needs testing. But in general I don't see why we should run the controller in non-manual mode if we are landed and if we are not doing a takeoff.